docs/howto/write-and-structure-charm-code.md

```diff
@@ -269,6 +269,46 @@ def get_version() -> str:
     return response.text
 ```
 
+(handle-status)=
+## Handle status
+
+Your charm should report its status to Juju. The best approach depends on the charm's complexity:
+
+  - If units don't need to coordinate, use [`self.unit.status`](ops.Unit.status) to report the
+    status of each unit independently. For example:
+
+    ```python
+    def _on_start(self, event: ops.StartEvent):
+        """Handle start event."""
+        self.unit.status = ops.MaintenanceStatus("starting server")
+        # ...
+        # Start the workload and wait for it to be ready.
+        # ...
+        self.unit.status = ops.ActiveStatus()
+    ```
+
+    When your charm code sets `self.unit.status`, Ops immediately sends the unit status to Juju.
+
+    Juju aggregates the unit statuses into an overall application status.
```

Comment:
I was going to suggest that this link to Juju to explain how the status is chosen, but then when I found the relevant page I noticed that it's the same as the "see also" below. We probably don't want to link to the same page twice, even if one link is to a specific section. Any ideas on how to indicate that you'll find the info there? If not, I'm fine with this as-is.